services/api/routers/crud.py

```diff
@@ -0,0 +1,58 @@
+"""crud functions"""
+import pgeocode
+from sqlalchemy import text
+from sqlalchemy.orm import Session
+
+
+def zipcode_to_latlong(zipcode: str):
+    """helper func, returns tuple of lat long"""
+    geo = pgeocode.Nominatim('us')
+    loc = geo.query_postal_code(zipcode)
+    return loc["latitude"], loc["longitude"]
+
+def get_nearby_stations(zipcode: str, db: Session):
+    """given zipcode, returns the 5 nearest stations"""
+    loc = zipcode_to_latlong(zipcode)
+    stmt = text(
+        """
```

Comment:
might be that raw sql is easiest and best, but maybe we can talk about using sqlalchemy orm too?